src/content/en/updates/2020/05/devtools.md

```diff
@@ -0,0 +1,126 @@
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/39aac148144c5464a67e9013f3c218f6ea65dfdf #}
+
+Chromium Bug: [#1068116](https://crbug.com/1068116)
+
+## Performance panel updates
+
+### Total Blocking Time information in the footer {: #tbt }
+
+After recording your load performance, the Performance panel now shows
+[Total Blocking Time](https://web.dev/tbt/) (TBT) information in the footer.
+TBT is a load performance metric that helps quantify how long it takes a page
+to become usable. It essentially measures how long a page *appears* to be usable
+(because its content has been rendered to the screen) but *isn't actually usable*
+because JavaScript is blocking the main thread and therefore the page can't respond
+to user input. TBT is the main [lab metric](https://web.dev/how-to-measure-speed/#lab-data-vs-field-data)
+for approximating First Input Delay, which is one of Google's new
+[Core Web Vitals](https://web.dev/vitals/#core-web-vitals).
+
+{# TODO(kayce): Get a working screenshot of the UI. }
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/a0c450d5b58f71b67134306b2e1c29a75326d3db #}
+
+Chromium Bug: [#1054381](https://crbug.com/1054381)
+
+<!--
+
+### Cumulative Layout Shift (CLS) movement records {: #cls }
+
+{# TODO(kayce): Get a working screenshot of the UI. }
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/f130314cc50ccd2c933017ee9bb06eefd68ed206 #}
+
+-->
+
+## Styles pane updates {: #styles }
+
+### Support for the `revert` keyword {: #revert }
+
+[revert]: https://developer.mozilla.org/en-US/docs/Web/CSS/revert
+[revert-bcd]: https://developer.mozilla.org/en-US/docs/Web/CSS/revert#Browser_compatibility
+
+The Styles pane's autocomplete UI now detects the [`revert`][revert] CSS keyword, which
+reverts the cascaded value of a property to what the value would have been if no changes
+had been made to the element's styling.
+
+![Setting the value of a property to `revert`](/web/updates/images/2020/05/revert.jpg)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/811c6e98ee25baf3f6f5a6145e1896fbe4362d7a #}
+
+Chromium Bug: [#1075437](https://crbug.com/1075437)
+
+### Image previews {: #image-previews }
+
+Hover over a `background-image` value in the Styles pane to see a preview of
+the image in a tooltip.
+
+![Hovering over a `background-image` value](/web/updates/images/2020/05/image-preview.jpg)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/1ebbb30be21f07d6a4d3ccd3b8dc7bd0dd37b41d #}
+
+Chromium Bug: [#1040019](https://crbug.com/1040019)
+
+### Support for space-separated functional color notations {: #functional-color-notation }
+
+[CSS Color Module Level 4](https://www.w3.org/TR/css-color-4/#changes-from-3)
+specifies that color functions like `rgb()` should support space-separated
+arguments. For example, `rgb(0, 0, 0)` is equivalent to `rbg(0 0 0)`. The Styles
+pane now recognizes the space-separated syntax.
```

Comment:
It already recognized it before. What's new is that it now *produces* the modern syntax by default, instead of the deprecated legacy syntax. The reason for that is <https://twitter.com/mathias/status/1253242715304857601>: 

> Upcoming new features such as `lab()`, `lch()`, and `color()` use the same syntax (and don’t work with commas).

Modern browsers have supported the new syntax [for a long time](https://caniuse.com/#feat=mdn-css_types_color_space_separated_functional_notation).